docs/csharp/language-reference/keywords/false-literal.md

```diff
@@ -1,21 +1,25 @@
 ---
-title: "false Literal (C# Reference)"
+title: "false literal (C# Reference)"
 ms.date: 07/20/2015
 helpviewer_keywords: 
   - "false literal keyword [C#]"
 ms.assetid: fcc1c1a5-0e42-4226-a128-81492a9bf719
 ---
-# false Literal (C# Reference)
+# false literal (C# Reference)
+
 Represents the boolean value false.  
+
+## Example
+
+[!code-csharp[csrefKeywordsOperator#3](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsOperator/CS/csrefKeywordsOperators.cs#3)]
+
+## C# language specification
+
+[!INCLUDE[CSharplangspec](~/includes/csharplangspec-md.md)]  
   
-## Example  
- [!code-csharp[csrefKeywordsOperator#3](../../../csharp/language-reference/keywords/codesnippet/CSharp/false-literal_1.cs)]  
-  
-## C# Language Specification  
- [!INCLUDE[CSharplangspec](~/includes/csharplangspec-md.md)]  
-  
-## See Also  
- [C# Reference](../../../csharp/language-reference/index.md)  
- [C# Programming Guide](../../../csharp/programming-guide/index.md)  
- [C# Keywords](../../../csharp/language-reference/keywords/index.md)  
- [true](../../../csharp/language-reference/keywords/true.md)
+## See also
+
+[C# Reference](../../../csharp/language-reference/index.md)  
```

Comment:
not related to your PR: Consider simplifying the paths.

Reply:
yes, I was on the fence on whether I did this with this change or not. I'll start doing this as part of these changes then.